x-pack/filebeat/module/zeek/README.md

```diff
@@ -0,0 +1,73 @@
+# Zeek (Bro) module
+
+## Caveats
+
+* Module is to be considered _alpha_.
```

Comment:
We don't have a notion of Alpha in Elastic, we use either Beta or Experimental. Both Beta and Experimental are not recommended for production and our support team accepts Sev-3 issues only. The difference between them is that for Beta we have clear plans to move them to GA, while Experimental features might go away if unsuccessful. In this case, I'd say Beta fits best.